autoload/battery/backend/termux.vim

```diff
@@ -19,7 +19,7 @@ endfunction
 
 function! s:on_exit(backend, buffer, exitval) abort
   let content = join(a:buffer, '')
-  let a:backend.is_charging = json_decode(content).status !=# 'DISCHARGING'
+  let a:backend.is_charging = json_decode(content).status !=# 'NOT_CHARGING' || 'DISCHARGING'
```

Comment:
This fix doesn't make sense. What did you mean?

Reply:
As i was scrolling here i see that the first person who PR this feature use NOT_CHARGING so i add the or statement

Reply:
It seems like my understanding on vim script is zero, can you help me? Or just reject this PR if you think its just a bother for you, I'm sorry for taking your time

I can't seem to understand how to check if one of those string is exist